src/trip.core.js

```diff
@@ -507,7 +507,7 @@ Trip.prototype = {
     this.setTripBlock(o);
     this.showTripBlock(o);
 
-    if (this.settings.enableAnimation) {
+    if (TripUtils.isSet(o.enableAnimation, this.settings.enableAnimation)) {
```

Comment:
Yes i know, but in order to make the DOM reflect its real state, I still prefer to remove that animation class ever time  ! No matter how, let me merge this first, thanks a lot !!!